subsys/net/lib/download_client/src/download_client.c

```diff
@@ -65,8 +72,11 @@ static int socket_timeout_set(int fd, int type)
 		.tv_usec = (timeout_ms % 1000) * 1000,
 	};
 
+#if defined (CONFIG_POSIX_API)
```

Comment:
From the checkpatch report:
```suggestion
#if defined(CONFIG_POSIX_API)
```

Reply:
fixed